tools/ci_build/github/linux/docker/Dockerfile.manylinux2010_gpu

```diff
@@ -7,7 +7,6 @@ ENV NVIDIA_DRIVER_CAPABILITIES compute,utility
 
 ADD scripts /tmp/scripts
 RUN cd /tmp/scripts && /tmp/scripts/install_centos.sh && /tmp/scripts/install_deps.sh -p $PYTHON_VERSION && \
-yum remove -y devtoolset-\*  && yum install -y devtoolset-7-binutils devtoolset-7-gcc devtoolset-7-gcc-c++ devtoolset-7-gcc-gfortran devtoolset-7-libquadmath-devel devtoolset-7-libstdc++-devel devtoolset-7-runtime && \
```

Comment:
should tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_gpu be updated too?

Reply:
Where it is used?

Reply:
manylinux2014 is a different story. I guess its gcc version is higher than the one(8.3) in manylinux 2010, and CUDA 10.1 may not work with it.

Reply:
it's used by the training python package build